components/camera/camera_waitgroup.go

```diff
@@ -0,0 +1,48 @@
+package camera
+
+import (
+	"context"
+	"sync"
+
+	"github.com/edaniels/gostream"
+
+	"go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/rimage/transform"
+)
+
+// WaitGroupCamera is a wrapper for camera.Camera with a sync.WaitGroup.
+type WaitGroupCamera struct {
```

Comment:
Renamed to avoid stutter error from linter.

```
exported: type name will be used as camera.CameraWaitGroup by other packages, and that stutters; consider calling this WaitGroup (revive)
```